tests/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "osio-testrunner",
+  "version": "0.0.1",
+  "description": "",
+  "keywords": [
+    "angular2",
+    "fabric8",
+    "openshift.io"
+  ],
+  "author": "Michael Kleinhenz <kleinhenz@redhat.com>",
+  "license": "Apache 2",
+  "scripts": {
+    "tsc": "tsc",
+    "webdriver:update": "webdriver-manager update",
+    "webdriver:start": "webdriver-manager start",
+    "protractor": "protractor tmp/protractor.conf.js",
+    "test": "./run.sh"
+  },
+  "dependencies": {
+    "@types/jasmine": "2.8.5",
+    "jasmine": "2.9.0",
+    "jasmine-spec-reporter": "4.2.1",
+    "npm-run-all": "^4.1.2",
```

Comment:
I don't see where this is being used

Reply:
The jasmine-spec-reporter is needed to get the nice output formatting, but the other stuff can go, yes.